config: document new `core.autostage` setting (#1779)
iterative/dvc.org: content/docs/command-reference/add.md

5. Add the `targets` to `.gitignore` in order to prevent them from being
   committed to the Git repository (unless `dvc init --no-scm` was used when
   initializing the DVC project).

> 6. If [DVC Metafiles](/doc/user-guide/dvc-files-and-directories) are modified,
>    DVC will print instructions showing `git` commands for adding the changes (or
>    stage them automatically if
>    [`core.autostage`](/doc/command-reference/config#core) is set).

Reviewer: `add` only uses .dvc files
```suggestion
6. Instructions are printed showing `git` commands for staging `.dvc` files
  (or they are staged automatically if
   [`core.autostage`](/doc/command-reference/config#core) is set).
```

5. Add the `targets` to `.gitignore` in order to prevent them from being
   committed to the Git repository (unless `dvc init --no-scm` was used when
   initializing the DVC project).
6. Instructions are printed showing `git` commands for staging `.dvc` files
  (or they are staged automatically if

>    [`core.autostage`](/doc/command-reference/config#core) is set).

Reviewer: This is the perfect place for this note, thanks!

I wonder if other there are similar places in other commands that need this same parenthesis...

Reviewer: Please if you see any other places, add this in another PR. Thanks
